src/lib/webpack.ts

```diff
@@ -17,6 +17,8 @@ export function createWebpackCompiler(
 
 		// https://stackoverflow.com/a/64715069
 		publicPath: '',
+		// to override in possible custom paths in the webpack config
+		filename: '[name].js',
```

Comment:
Hadn't considered that. Good catch!

Reply:
I don't think it matters much in the end since the code is adding the entry points itself, but it can be sure.

Reply:
Rebased and changed the commit to just use main.js since I was removing the memfs change I made due to #15